### PR TITLE
Find Doctor

### PR DESCRIPTION
**Pull request by kimshitong**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindDoctorCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import seedu.address.model.person.NameContainsKeywordsDoctorPredicate;
+
+
+/**
+ * Finds and lists all doctors in database whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindDoctorCommand extends NewCommand {
+
+    public static final String COMMAND_WORD = "find";
```
Comment (yanghengtang) [inline src/main/java/seedu/address/logic/commands/FindDoctorCommand.java:19]: Change to "find_d"

File: src/main/java/seedu/address/logic/commands/ListDoctorCommand.java
Comment (yanghengtang) [inline src/main/java/seedu/address/logic/commands/ListDoctorCommand.java:1]: Can check why is ListDoctorCommand added in "Find Doctor PR"?

**Review by yanghengtang — COMMENTED** [review]
It seems like ListDoctorCommand and its related classes are added into this PR as well. Any reason for that?

File: src/main/java/seedu/address/logic/commands/ListDoctorCommand.java
Comment (kimshitong) [reply src/main/java/seedu/address/logic/commands/ListDoctorCommand.java:1]: Uhh, this was my bad... I thought it needed, so I created a new branch on the extend of Find Doctor PR.

I can create a new branch and a new pull request, no worries.

File: src/main/java/seedu/address/logic/commands/FindDoctorCommand.java (hunk 1 above)
Comment (kimshitong) [reply src/main/java/seedu/address/logic/commands/FindDoctorCommand.java:19]: Ok, thanks

**Pull request closed without merging**